Dockerfile

```diff
@@ -8,7 +8,7 @@ RUN R -q -e 'install.packages(c("shinyFeedback", "shinyWidgets", "shinydashboard
 
 RUN R -q -e "remotes::install_github('getwilds/proofr@v0.2')"
 
-RUN R -q -e "remotes::install_github('getwilds/rcromwell@v3.2.1')"
+RUN R -q -e "remotes::install_github('getwilds/rcromwell@dev')"
```

Comment:
@sckott -- Now that the cromwell_labels function has been merged into main, did you want to point this to main? Or did you want to version the repo first and point to that?

Reply:
https://github.com/getwilds/rcromwell/pull/47 has been merged and a new release has been minted for the `rcromwell` repo: https://github.com/getwilds/rcromwell/releases/tag/v3.2.5

Reply:
great, okay, pushed change to latest rcromwell